docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+az acr supply-chain workflow create -r myRegistry -g myResourceGroup -t continuouspatchv1 -–config ./continuouspatching.json --schedule 1d –-dry-run   
+```
+
+The --dry-run flag will output all specified artifacts by the JSON file configuration. Customers can verify that the right artifacts are selected. With the sample ubuntu configuration above, the following results should be displayed as output. 
+```sh
+Ubuntu: jammy-20240111
+Ubuntu: jammy-20240125
+```
+Help command to see all required/optional flags.
+```sh
+az acr supply-chain workflow create --help
+```
+ 
+5. Once satisfied with the dry-run results, run the ‘create’ command again without the --dry-run flag to officially create your continuous patching workflow.  
+
+**Important**
+
+The --schedule parameter follows a fixed-day multiplier starting from day 1 of the month. This means:
+
+- If you specify --schedule 7d and run the command on the 3rd, the next scheduled run will be on the 7th—because 7 is the first multiple of 7 (days) after the 3rd, counting from day 1 of the month.
+
+- If --schedule is 3d and today is the 7th, then the next scheduled run lands on the 9th—since 9 is the next multiple of 3 that follows 7.
+
+- If you add the flag --run-immediately, you trigger an immediate patch run. The subsequent scheduled run will still be aligned to the nearest day multiple from the 1st of the month, based on your --schedule value.
+
+Command Schema:
+```sh
+az acr supply-chain workflow create -r <registryname> -g <resourcegroupname> -t continuouspatchv1 -–config <JSONfilename> --schedule <number of days> --run-immediately
+```
+
+Example Command: 
+```sh
+az acr supply-chain workflow create -r myRegistry -g myResourceGroup -t continuouspatchv1 -–config ./continuouspatching.json --schedule 1d --run-immediately
+```
+
+Upon a successful command (whether or not you include --run-immediately), you will see:
+
+- A success message confirming that your workflow tasks have been queued.
+
+- An output parameter indicating when the next run of your workflow is scheduled, so you can track exactly when patching will occur again.
+
+Help command to see all required/optional flags.
+```sh
+az acr supply-chain workflow create --help
+```
+## Use Azure Portal to view workflow tasks
+
+Once the workflow succeeds, go to the Azure Portal to view your running tasks. Click into Services -> Repositories, and you should see a new repository named “csscpolicies/patchpolicy”. This repository hosts the JSON configuration artifact that will be continuously referenced for continuous patching.  
+
+![PortalRepos](./media/portal_repos1.png)
+
+Next, click on “Tasks” under “Services”. You should see 3 new tasks, named the following:
+
+![PortalTasks](./media/portal_tasks1.png)
+
+- cssc-trigger-workflow – this task scans the configuration file and calls the scan task on each respective image.    
+- cssc-scan-image – this task scans the image and calls the patching task **only if vulnerabilities were found**.
```

Comment:
Please clarify that the scan task will only call the patching task when the 2 conditions are met (OS vulnerabilities rather than just vulnerabilities, and an image not considered EOSL).

- cssc-scan-image – this task scans the image for operating system vulnerabilities. This task will only trigger the patching task only if (1) operating system vulnerabilities were found, and (2) the image is not considered End of Service Life (EOSL). For more information on EOSL, please consult [Preview Limitations](#preview-limitations).